app/src/main/java/org/simple/clinic/search/PatientSearchScreenController.kt

```diff
@@ -199,8 +110,9 @@ class PatientSearchScreenController @Inject constructor(
 
     return Observables.combineLatest(searchClicks, validationErrors)
         .filter { (_, errors) -> errors.isEmpty() }
-        .withLatestFrom(nameChanges, ageChanges, dateOfBirthChanges) { _, name, age, dob -> Triple(name, age, dob) }
-        .map { (name, age, dateOfBirth) -> { ui: Ui -> ui.openPatientSearchResultsScreen(name, age, dateOfBirth) } }
+        .withLatestFrom(nameChanges) { _, name ->
+          { ui: Ui -> ui.openPatientSearchResultsScreen(name) }
+        }
   }
 
   private fun saveAndProceeds(events: Observable<UiEvent>): Observable<UiChange> {
```

Comment:
Looks like this function is incorrectly named. Can you please rename this to something like `openPatientRegistrationScreen()` or something as part of this PR?

Reply:
We could also skip this if you want.

Reply:
The name seems to be correct, as it opens the `PatientSearchResultsScreen`.